server/text_generation_server/models/causal_lm.py

```diff
@@ -86,6 +85,7 @@ def move_data(dst_tensor, chunk_size, indices, src_tensors):
     result = dst_tensor
     if chunk_size > 1:
         dst_tensor = dst_tensor.view(bs // chunk_size, chunk_size, *dst_tensor.shape[1:])
+    import habana_frameworks.torch as htorch
```

Comment:
why importing it every time it is called? it is redundant

Reply:
yes, probably could be better, but it shouldn't be imported in global scope - we need to setup env. variables before habana_frameworks is loaded (fp8 synapse requirement)

Reply:
This is very error prone and tedious...